### PR TITLE
add BulkUpdateStatusTestAsync

### PR DESCRIPTION
**Pull request by alesyalis**

### CODE REVIEW

File: Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs [hunk 1]
```diff
@@ -143,6 +143,21 @@ public Task UpdateStatusByIdsAsync(IEnumerable<Guid> ids, Guid projectId, Status
             return GetCollection().UpdateManyAsync(filter, updateDefinition, updateOptions);
         }
 
+        public Task BulkUpdateStatusTestAsync(IEnumerable<Guid> candidatesIds, Guid projectId, Status status)
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs:146]: Remove word Test, it's universal method for different Statuses

File: Recrutify/Recrutify.Services/Services/CandidateService.cs [hunk 2]
```diff
@@ -217,6 +217,8 @@ public async Task BulkSendEmailsWithTestAsync(BulkSendEmailWithTestDTO bulkSendE
         {
             var candidates = await GetCandidatesByIdsAsync(bulkSendEmailWithTestDTO.CandidatesIds);
             var project = await _projectService.GetAsync(projectId);
+            var status = Status.Test;
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Services/Services/CandidateService.cs:220]: Not necessary to create var, can just pass Status.Test to repository

Comment (alesyalis) [reply Recrutify/Recrutify.Services/Services/CandidateService.cs:220]: done

File: Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs (hunk 1 above)
Comment (alesyalis) [reply Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs:146]: done

**Review by elizavetapavliv — APPROVED** [review]

**Pull request merged**